web/src/pages/blog/index.astro

```diff
@@ -0,0 +1,15 @@
+---
+import MainLayout from "../../layouts/global.astro"
+import PostCard from "../../components/post.astro"
+
+const allPosts = await Astro.glob("./*.mdx")
+---
+
+<MainLayout title="Blog">
+	<section class="container" aria-label="New Blog Posts">
+		<h1 class="h1">New Blog Posts</h1>
+		<div class="post-container">
```

Comment:
Same, these classes don't actually do anything without a corresponding entry in `global.css` which you shouldn't do with tailwind.